language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/SelectClauseNodeContext.java

```diff
@@ -89,4 +118,32 @@ public boolean onPreValidation(BallerinaCompletionContext context, SelectClauseN
         return !node.selectKeyword().isMissing() && 
                 context.getCursorPositionInTree() >= node.selectKeyword().textRange().startOffset();
     }
+    
+    private List<FunctionSymbol> getLangLibMethods(BallerinaCompletionContext context) {
+
+        Types types = context.currentSemanticModel().get().types();
+        List<FunctionSymbol> langLibMethods = types.INT.langLibMethods();
+        langLibMethods.addAll(types.DECIMAL.langLibMethods());
+        langLibMethods.addAll(types.FLOAT.langLibMethods());
+        langLibMethods.addAll(types.STRING.langLibMethods());
+        langLibMethods.addAll(types.BOOLEAN.langLibMethods());
+        langLibMethods.addAll(types.ANY.langLibMethods());
+        langLibMethods.addAll(types.ANYDATA.langLibMethods());
+        langLibMethods.addAll(types.BYTE.langLibMethods());
+        langLibMethods.addAll(types.ERROR.langLibMethods());
+        langLibMethods.addAll(types.FUNCTION.langLibMethods());
+        langLibMethods.addAll(types.FUTURE.langLibMethods());
+        langLibMethods.addAll(types.HANDLE.langLibMethods());
+        langLibMethods.addAll(types.JSON.langLibMethods());
+        langLibMethods.addAll(types.NEVER.langLibMethods());
+        langLibMethods.addAll(types.NIL.langLibMethods());
+        langLibMethods.addAll(types.READONLY.langLibMethods());
+        langLibMethods.addAll(types.REGEX.langLibMethods());
+        langLibMethods.addAll(types.STREAM.langLibMethods());
+        langLibMethods.addAll(types.TYPEDESC.langLibMethods());
+        langLibMethods.addAll(types.XML.langLibMethods());
+        langLibMethods.addAll(types.COMPILATION_ERROR.langLibMethods());
```

Comment:
Is there specific  reason for adding this

Reply:
The aim was to add all the types in the TypesAPI. Can remove if it's not necessary.

Reply:
This is not necessary IMO. This type is for internal use only(LS). This should not be visible to the VSCode user. Therefore we shouldn't suggest langlibs for that.